pkg/services/export/git_export_job.go

```diff
@@ -180,18 +181,29 @@ func (e *gitExportJob) process(exporters []Exporter) error {
 			continue
 		}
 
+		e.status.Target = exp.Key
+		e.helper.exporter = exp.Key
+
+		before := e.helper.counter
 		if exp.process != nil {
-			e.status.Target = exp.Key
-			e.helper.exporter = exp.Key
 			err := exp.process(e.helper, e)
+
 			if err != nil {
 				return err
 			}
 		}
 
 		if exp.Exporters != nil {
-			return e.process(exp.Exporters)
+			err := e.process(exp.Exporters)
+			if err != nil {
+				return err
+			}
 		}
+
+		prev := e.status.Count[exp.Key]
+		e.status.Count[exp.Key] = prev + (e.helper.counter - before)
```

Comment:
```suggestion
		e.status.Count[exp.Key] += e.helper.counter - before
```

Reply:
does this mean we can have more than one exporter with the same key?

Reply:
I added a comment to make it more clear what is happening.  This process gets called independently for each org, this report aggregates all the orgs stats into one report.

It may be better to report each org independently, but that will require a slightly more involved refactoring